packages/hoppscotch-app/components/lenses/renderers/composables/useCopyResponse.ts

```diff
@@ -0,0 +1,22 @@
+import { Ref, ref } from "@nuxtjs/composition-api"
+import { copyToClipboard } from "~/helpers/utils/clipboard"
+
+export default function useCopyResponse(
+  responseBodyText: Ref<any>,
+  toast: any,
+  t: any
```

Comment:
These are derivable from inside the composable, do not pass these down and refrain from using any.